src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/NoProvidersAlert.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function NoProvidersAlert({ currentLocation, typeOfCareName }) {
```

Comment:
You may want to have this component receive `sortMethod` instead of `currentLocation`.  It's possible the user can get their current location, then switch the sort method back to residential, resulting in incorrect copy in the conditional below.

Reply:
Rgr, fixed.